src/open_inwoner/scss/components/Header/PrimaryNavigation.scss

```diff
@@ -98,4 +98,9 @@
   &:not(#{&}--dismissed) > &__list > &__list-item:hover &__list {
     transform: scaleY(1);
   }
+
+  &__list--hide {
```

Comment:
is this style used anywhere? (not if we do not use the primary-navigation JavaScript from this PR?).

Reply:
No, it is not needed now. I will remove it.